app/models/concerns/illiadable.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# Mixin for requests that should be sent to ILLiad for initial handling
+module Illiadable
+  def submit!
```

Comment:
Using modules that override methods on the class they are mixed into, isn't the easiet to read. Is there a reason this is needed? I see that HoldRecall#submit! already calls SubmitIlliadRequestJob can we modify Scan#submit to call SubmitIlliadRequestJob directly?

even better might be to convert this module to a new class (e.g. IlliadRequest) and have a method on the "Illiadable" requests like #illiad_request. I think there is value in separating the concept of "The user made a request" and "We made this ILL request to fulfull the users request"

Reply:
I'm fine just not including this method in the mixin – I figured if you mixed in Illiadable it'd be because you want to submit the request via ILLiad, but in the case of scans we _also_ want to do something else, so it's fine to just be explicit instead.

Reply:
Updated to remove the method and make `Scan#submit!` explicitly call the illiad job